### ISSUE TITLE
[$20][MSFT-54] [Register] Member registration page doesn't have titles that describe topic or purpose.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2771]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member/registration
2. Check the page title

**Expected Result**
Web pages have titles that describe topic or purpose. Should display as Register | Topcoder

**Actual Result**
Member registration page doesn't have titles that describe topic or purpose.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 3. Understandable
**WCAG Level**: A
**WCAG Success Criterion**: 2.4.2 Page Titled
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F30

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096316 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096316 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096316 has been updated - it has been assigned to Timor.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096316 has been updated - it has been assigned to gets0ul.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu @gets0ul I guess it is already completed by one of my PR. will confirm you shortly

**Comment by gets0ul** [comment]
@PrakashDurlabhji @veshu 
Can you put the link of the PR you mentioned?

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2771 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2771 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2771 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by gets0ul** [comment]
@PrakashDurlabhji @veshu   
So how shortly is shortly?  It's been 12 hours since you left the comment.
I really don't want to step into other people's work.

I've been looking into the PRs you made
https://github.com/appirio-tech/accounts-app/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3APrakashDurlabhji+

I can not find anything that related with this issue.
The only thing that comes close is this.
 https://github.com/appirio-tech/accounts-app/pull/228
  
Reviewed by @mishacucicea   

It's for the Login page, which seems it does not solve the actual problem because it just changed the title from 'Topcoder' into 'Login | Topcoder' for ALL pages.

So the actual problem is still there, before it was 'Topcoder' for all pages, now we got 'Login | Topcoder' for all pages.

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097505 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097505 has been updated - it has been assigned to gets0ul.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by veshu** [comment]
@gets0ul Please fix, If it is duplicate fixes we won't merge it(though you will get paid for this if he hasn't referenced this issue on the PR he is claiming before you picked it up)

**Comment by gets0ul** [comment]
@veshu @mishacucicea 

PR: https://github.com/appirio-tech/accounts-app/pull/263
Also covers for https://github.com/topcoder-platform/community-app/issues/2762
as it does more proper way to handle dynamic page title, instead of hardcoded one as it's currently done now.

It sets up the base for configuring title for each page so it will be easier whenever specific title is needed for other page in future.

**Comment by nkumar-topcoder** [comment]
Node #2762 is already pushed to prod and verified, but no way this conflicts but extends page title

**Comment by gets0ul** [comment]
@nkumar-topcoder 
As I said before, that fix is done in not quite proper way
as 

> which seems it does not solve the actual problem because it just changed the title from 'Topcoder' into 'Login | Topcoder' for ALL pages.
> 
> So the actual problem is still there, before it was 'Topcoder' for all pages, now we got 'Login | Topcoder' for all pages.

https://github.com/appirio-tech/accounts-app/pull/263#issuecomment-515306438

Can I get a little bump on the prize for all the reasons I have stated previously?
Thanks.

**Comment by drasticdpk** [comment]
Operating System: Windows
Browser: Chrome
https://accounts.topcoder-dev.com/member/registration

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097505 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by drasticdpk** [comment]
Verified on.
Operating System: Windows
Browser: Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097505<br/><br/>```This is an automated message for crazyk via Topcoder X```
